Rhdevdocs 6204: Creating a separate release note folder and a tile for docs.redhat.com for Builds 1.2 (#83741)
openshift/openshift-docs: modules/ob-release-notes-1-2.adoc

// This module is included in the following assemblies:
// * about/ob-release-notes.adoc

:_mod-docs-content-type: REFERENCE

> [id="ob-release-notes-1-1_{context}"]

Reviewer: ```suggestion
[id="ob-release-notes-1-2_{context}"]
```
This file needs changing everything from 1.1 to 1.2. I know you showed me those changes so maybe just the file did not get saved or something :smile:

= Release notes for {builds-shortname} General Availability 1.1

> {builds-shortname} General Availability (GA) 1.1 is now available on {ocp-product-title} 4.16.

Reviewer: ```suggestion
{builds-shortname} General Availability (GA) 1.2 is now available on {ocp-product-title} 4.16.
```

> {builds-shortname} General Availability (GA) 1.1 is now available on {ocp-product-title} 4.16.

Reviewer: ```suggestion
{builds-shortname} General Availability (GA) 1.1 is now available on {ocp-product-title} <TBD>.
```
As I suppose the version for OCP might change

> [id="new-features-1-1_{context}"]

Reviewer: ```suggestion
[id="new-features-1-2_{context}"]
```

== New features

> The following sections highlight what is new in {builds-shortname} 1.1.

Reviewer: ```suggestion
The following sections highlight what is new in {builds-shortname} 1.2.
```

> [id="builds-new-features-1-1_{context}"]

Reviewer: ```suggestion
[id="builds-new-features-1-2_{context}"]
```

=== Builds

> * The builds controllers now use Tekton's V1 API to create and access the `TaskRun` that backs a `BuildRun`.
> 
> * With this release, you can now define a build without any source. This is useful if you want to run the build using only the local source.
> 
> * The output image section now supports an optional timestamp field to change the image creation timestamp. Use the `SourceTimestamp` string to set it to match the source timestamp.
> 
> * With this release, the `.source.type` field is now required in both `build` and `buildRun`.
> 
> * With this release, the Operator now installs Shipwright by default and introduces several user experience enhancements for Builds.

Reviewer: You cn delete the actual RNs and just keep this as a template for when the actual RNs will come :) 
```suggestion

```

> [id="csi-driver-new-features-1-1_{context}"]

Reviewer: ```suggestion
[id="csi-driver-new-features-1-2_{context}"]
```

=== Shared Resource CSI Driver

> * The Shared Resource CSI Driver is now generally available. With this release, the CSI Shared Driver enables sharing of `ConfigMaps` and secrets across different namespaces in Kubernetes clusters. The driver has permission to read all Kubernetes secrets. This feature improves resource efficiency by reducing duplication and simplifies configuration management in multi-tenant environments. Administrators can define access policies to control which namespaces can read or modify these shared resources.

Reviewer: ```suggestion
```
Same as previous comment

> [id="known-issues-1-1_{context}"]

Reviewer: ```suggestion
[id="known-issues-1-2_{context}"]
```

== Known issues

> The following section highlights known issues in {builds-shortname} 1.1.

Reviewer: ```suggestion
The following section highlights known issues in {builds-shortname} 1.2.
```

> [id="builds-known-issues-1-1_{context}"]

Reviewer: ```suggestion
[id="builds-known-issues-1-2_{context}"]
```

=== Builds

> * With this release, direct upgrades from Builds 1.0 to Builds 1.1 are not supported. To upgrade to Builds 1.1, you must delete the `ShipwrightBuild` object, uninstall the Builds `1.0.z` operator, and then install the Builds `1.1.0` operator from OperatorHub.

Reviewer: ```suggestion
```
